### PR TITLE
Hot to do ACR + Managed-Identities on App Service

### PR DESCRIPTION
**Pull request by btardif**

### CODE REVIEW

**Comment by btardif** [comment]
FYI @aklopfer @calvinsID @panchagnula @nertim

File: HowTo/use_system-assigned_managed_identities.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+#URL for ACR instance create in STEP 0
+ACR_URL=$(az acr show -g $RG_Name --n $ACR_Name --query loginServer --output tsv)
+
+#Resource ID for WebApp configuration.
+Webapp_Config = $(az webapp show -g $RG_Name -n $Web_Name --query id --output tsv) + "/config/web"
+
+Image ="myapp:latest" #Docker image and tag to pull
+
+FX_Version = '"Docker|"' + $ACR_URL + "/" + $Image
```
Comment (aklopfer) [inline HowTo/use_system-assigned_managed_identities.md:83]: Do we need the double quotes around Docker| here?

**Review by aklopfer — APPROVED** [review]

**Review by aklopfer — APPROVED** [review]

**Comment by btardif** [comment]
FYI @jeffhollan

File: HowTo/use_system-assigned_managed_identities.md (hunk 1 above)
Comment (btardif) [reply HowTo/use_system-assigned_managed_identities.md]: Fixed this. I was originally writing and testing this on my windows laptop + powershell console, switched over to mac + zsh and cleanup the script

**Pull request merged**